itou/www/apply/views/process_views.py

```diff
@@ -205,11 +212,7 @@ def details_for_prescriber(request, job_application_id, template_name="apply/pro
 
     # Latest GEIQ diagnosis for this job seeker created by a *prescriber*
     geiq_eligibility_diagnosis = (
-        job_application.to_company.kind == CompanyKind.GEIQ
-        and GEIQEligibilityDiagnosis.objects.valid()
-        .filter(job_seeker=job_application.job_seeker, author_prescriber_organization__isnull=False)
-        .select_related("author", "author_geiq", "author_prescriber_organization")
-        .first()
+        job_application.to_company.kind == CompanyKind.GEIQ and _get_geiq_eligibility_diagnosis(job_application, True)
```

Comment:
```suggestion
        job_application.to_company.kind == CompanyKind.GEIQ and _get_geiq_eligibility_diagnosis(job_application, only_prescriber=True)
```
(et pareil pour les suivants :sweat_smile: )

Reply:
à vrai dire j'ai hésité à le mettre partout ou nulle part ^^'
Je change partout alors